rel-eng/atomic-enterprise-master.service

```diff
@@ -3,16 +3,14 @@ Description=Atomic Enterprise Master
 Documentation=https://github.com/openshift/origin
 After=network.target
 After=etcd.service
-Before=atomic-enterpries-node.service
-Before=atomic-enterprise-sdn-node.service
-Before=atomic-enterprise-sdn-master.service
+Before=atomic-enterprise-node.service
 Requires=network.target
 
 [Service]
 Type=notify
-EnvironmentFile=/etc/sysconfig/atomic-enterprise-master
+EnvironmentFile=/etc/sysconfig/origin-master
 Environment=GOTRACEBACK=crash
-ExecStart=/usr/bin/atomic-enterprise start master --config=${CONFIG_FILE} $OPTIONS
+ExecStart=/usr/bin/openshift start master --config=${CONFIG_FILE} $OPTIONS
```

Comment:
Ok, there's a lot of symlinks we weren't providing. I'll add all of these minus the legacy `osc` and `osadm` ?

```
openshift-router       
openshift-deploy       
openshift-sti-build    
openshift-docker-build 
origin                 
atomic-enterprise      
osc                    
oadm                   
osadm                  
kubectl                
kubernetes             
kubelet                
kube-proxy             
kube-apiserver         
kube-controller-manager
kube-scheduler         
```